app/models/site_config.rb

```diff
@@ -46,6 +46,7 @@ class SiteConfig < RailsSettings::Base
   field :logo_png, type: :string, default: "https://practicaldev-herokuapp-com.freetls.fastly.net/assets/devlogo-pwa-512.png"
   field :logo_svg, type: :string, default: ""
   field :primary_sticker_image_url, type: :string, default: "https://practicaldev-herokuapp-com.freetls.fastly.net/assets/rainbowdev.svg"
+  field :onboarding_taskcard_sticker, type: :string, default: "https://practicaldev-herokuapp-com.freetls.fastly.net/assets/staggered-dev.svg"
```

Comment:
@maestromac we should probably be adding this new field to the Site config view too so that other communities can configure this value. I'm not sure what the scope of your task is but if it's not within the scope, you can probably add it as an issue in Team OSS's generalization pipeline, or ping me to add it :) .

Reply:
Thanks for pointing that out! I went ahead and implemented it. 

Re-requesting review